leapp/cli/upgrade/__init__.py

```diff
@@ -240,12 +240,14 @@ def upgrade(args):
         workflow.load_answers(answerfile_path, userchoices_path)
         workflow.run(context=context, skip_phases_until=skip_phases_until, skip_dialogs=True)
 
+    logger.info("Answerfile will be created at %s", answerfile_path)
```

Comment:
well at line 239 you have 'Using answerfile at '.. and here you write will be created kind of weird no?

Better saying 'Storing answerfile to ... instead since you might not 'create' it again, but just save the current state.